frontend-react/src/components/Table/TableFilters.tsx

```diff
@@ -18,23 +18,19 @@ export enum StyleClass {
     DATE_CONTAINER = "date-picker-container tablet:grid-col",
 }
 
-export enum FilterName {
-    START_RANGE = "start-range",
-    END_RANGE = "end-range",
-    CURSOR = "cursor",
-    SORT_ORDER = "sort-order",
-    PAGE_SIZE = "page-size",
-}
-
 interface SubmissionFilterProps {
     filterManager: FilterManager;
     cursorManager?: CursorManager;
+    onFilterClick?: any;
 }
 
 /* This helper ensures start range values are inclusive
  * of the day set in the date picker. */
 const inclusiveDateString = (originalDate: string) => {
```

Comment:
Let's export this function and add a couple of tests! 🙏

Reply:
This is already exported at the end of file. We currently have one test, I will ad another.